### ISSUE TITLE
Error code 401

### ISSUE THREAD

**Issue by Ozzymoto** [bspranger/Hubitat_iComfort#8]
i am new to hubitat, and i am working on getting my  Icomfort s30, i followed directions and get Error code 401...  please help

**Comment by bilaldabet** [comment]
any luck ? tried and same error

**Comment by Ozzymoto** [comment]
no, im so new i just dont get it... i was hoping someone would take me under there wing....

**Comment by Ozzymoto** [comment]
No, I pray something me will help! But it’s 4 years old so chance is slim.. also being so new to this I need step by step instructions as I appear to be lostBest regards Michael OsmerOn Nov 30, 2022, at 1:43 PM, Bilal ElDabet ***@***.***> wrote:﻿
any luck ? tried and same error

**Comment by Ozzymoto** [comment]
please for the love of god will someone help us!

**Comment by bilaldabet** [comment]
it is not compatible with S30 wifi hub. still trying to figure something out.

**Comment by Ozzymoto** [comment]
Wow, thank you so much for responding! Yes it appears I have the S30!Thank you so much for responding! I’m so new to this!I am also trying to get my SolarEdge production, and I’m just lost! Please please keep me in the loop if something is figured out for the S30  Mike O.Sent from Mail for Windows From: Bilal ElDabetSent: Thursday, December 1, 2022 2:25 PMTo: bspranger/Hubitat_iComfortCc: Ozzymoto; AuthorSubject: Re: [bspranger/Hubitat_iComfort] Error code 401 (Issue #8) it is not compatible with S30 wifi hub. still trying to figure something out.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
